specification/sql/resource-manager/readme.md

```diff
@@ -66,6 +66,7 @@ input-file:
 - Microsoft.Sql/2015-05-01-preview/syncMembers.json
 - Microsoft.Sql/2015-05-01-preview/virtualNetworkRules.json
 - Microsoft.Sql/2017-03-01-preview/cancelOperations.json
+- Microsoft.Sql/2017-03-01-preview/serverDnsAliases.json

```

Comment:
please also add to schema-2017-03-preview section

Reply:
This tag does not exist, so I have created it